MozillaFoundation/foundation.mozilla.org
# Remove mozfest schedule link

# Description

This PR removes the mozfest schedule link

Related PRs/issues: TP1-694 / #12371

# To Test
1) Pull branch locally
2) Navigate to mozfest.localhost:8000 and notice there is no longer a schedule link in the primary nav

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-997)
